### ISSUE TITLE
The wrong type for "refunds" is used in the "OrderManagementOrder" class

### ISSUE THREAD

**Issue by alexions** [klarna/kco_rest_dotnet#57]
we are using the .NET library and we found a bug:
In the class "OrderManagementOrder" the wrong type for "refunds" is used.

It should be like that:

```csharp
/// </summary>
[JsonProperty(PropertyName = "refunds")]
public ICollection<OrderManagementGetRefundResponse> Refunds { get; set; }
```

**Comment by alexions** [comment]
The PR is in the code-review, feel free to comment.

The initial issue:
For some reason we have 2 similar Refund models: OrderManagementRefund and OrderManagementGetRefundResponse. By fact - these two models are the same (based on the original documentation).

Ideal fix: Actualize one of the model and use it everywhere.
Real fix: In order to keep the backward compatibility I kept both of the models, but now the OrderManagementGetRefundResponse just inherits the OrderManagementRefund with no changes. Let's say - they are the same.

I actualized the OrderManagementRefund and put all the fields specified in the documentation.

So the issue must be fixed now.
